packages/docmanager-extension/src/index.tsx

```diff
@@ -916,12 +916,13 @@ function addCommands(
           ) {
             const result = await InputDialog.getText({
               title: trans.__('Rename file'),
-              okLabel: trans.__('Rename'),
+              okLabel: trans.__('Rename and Save'),
               placeholder: trans.__('File name'),
+              cancelLabel: trans.__('Save as %1', oldName),
```

Comment:
Not sure about this, but maybe we do not need to show the extension on the label?